website/content/docs/platform/k8s/vso/installation.mdx

```diff
@@ -5,15 +5,17 @@ description: >-
   The Vault Secrets Operator can be installed using Helm.
 ---
 
-# Installing the Vault secrets operator
+# Installing the Vault Secrets Operator
 
 ## Prerequisites
 
-- Kubernetes 1.22+
-- Vault 1.11+ or [HCP Vault](https://www.hashicorp.com/cloud)
```

Comment:
Should we keep this line? Seems helpful to say what version of vault we support and also that we support HCP.

Reply:
Nevermind, I found it.. :D